iguana/json_writer.hpp

```diff
@@ -50,7 +53,12 @@ IGUANA_INLINE void render_json_value(Stream &ss, T value) {
 template <typename Stream, string_container_t T>
 IGUANA_INLINE void render_json_value(Stream &ss, T &&t) {
   ss.push_back('"');
-  ss.append(t.data(), t.size());
+  if constexpr (fixed_array<T>) {
+    constexpr auto n = sizeof(T) / sizeof(decltype(std::declval<T>()[0]));
```

Comment:
std::extent_v can get array size, code will be simpler.

Reply:
ok, i get it